git_services/git_services/init/clone.py

```diff
@@ -15,9 +16,9 @@
     git_cloner = GitCloner(
         repositories=config.repositories,
         git_providers=config.git_providers,
-        workspace_mount_path=config.workspace_mount_path,
+        mount_path=config.mount_path,
```

Comment:
suggestion: workspace_mount_path was only ever used here, so now that we do not use it here we can remove it from the config.